.github/workflows/Java_CI.yaml

```diff
@@ -0,0 +1,16 @@
+name: Java CI
+
+on:
+  push:
+    branches: [ Feature/wrims-core ]
+
+jobs:
+  build:
+
+    runs-on: adoptopenjdk:8-jre-hotspot
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build with Gradle
```

Comment:
I recommend uploading the jar to the GitHub run so that you can inspect the output. See example: https://github.com/opendcs/rest_api/blob/main/.github/workflows/default.yml#L52